iroh-sync/src/store/fs.rs

```diff
@@ -115,13 +115,28 @@ pub struct Store {
     pubkeys: MemPublicKeyStore,
 }
 
+impl Default for Store {
+    fn default() -> Self {
```

Comment:
this is pretty confusing, I think  I would prefer

- `Store::memory()`
- `Store::persistent(path)`

similar to how I did it on the node builder.